commit c1014818af969ef33c67ca15ae3ff3b3bd204187 (refs/changes/11/4711/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-08-16T17:52:45-07:00 (1 year, 2 months ago)
    
    TRITON-681 node-imgmanifest support for requirements.bootrom (add changelog entry)

diff --git a/CHANGES.md b/CHANGES.md
index e77a2e1..b99ac00 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 (nothing yet)
 
+## 3.1.0
+
+- TRITON-681 Add optional 'requirements.bootrom' validation.
+
 ## 3.0.0
 
 - BREAKING change, DOCKER-929 Support registry 2.0 docker images.
diff --git a/Makefile b/Makefile
index f4778ba..1ba565a 100644
--- a/Makefile
+++ b/Makefile
@@ -46,6 +46,8 @@ versioncheck:
 	@echo version is: $(shell cat package.json | json version)
 	[[ `cat package.json | json version` == `grep '^## ' CHANGES.md | head -2 | tail -1 | awk '{print $$2}'` ]]
 
+check: versioncheck
+
 .PHONY: cutarelease
 cutarelease: versioncheck
 	[[ -z `git status --short` ]]  # If this fails, the working dir is dirty.
